sw/device/lib/dif/dif_pwrmgr.h

```diff
@@ -180,16 +180,19 @@ typedef struct dif_pwrmgr_wakeup_reason {
  * corresponding bit automatically, this function must be called before each
  * transition to low power state.
  *
- * Note: This function also syncs changes to the slow clock domain for them to
- * take effect.
+ * The corresponding CSR update needs to sync to the slow clock domain to
+ * take effect. This can be configured to skip it, but without synchronizing
```

Comment:
I think this should be "This function can be configured..." ? And similarly below.

Reply:
Done.